web/src/user_groups.ts

```diff
@@ -202,6 +202,23 @@ export function get_recursive_subgroups(target_user_group: UserGroup): Set<numbe
     return subgroup_ids;
 }
 
+export function get_recursive_group_members(target_user_group: UserGroup): Set<number> {
+    const members = new Set(target_user_group.members);
+    const subgroup_ids = get_recursive_subgroups(target_user_group);
+
+    if (subgroup_ids === undefined) {
+        return members;
```

Comment:
I could also add an early return when the subgroup id set size is 0, but there's no benefit to doing that since the for loop below is gonna get skipped anyways in that case and the next statement is `return members`

Reply:
It looks like `get_recursive_subgroups` can only return `undefined` if a subgroup was not defined? I'm not sure that's possible today, but probably this will need to be reworked if we add restrictions on viewing group membership.